release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -84,6 +84,10 @@ For more information, see xref:../installing/installing-troubleshooting.adoc#ins
 ==== Support for Microsoft Hyper-V generation version 2
 By default, the installation program now deploys a Microsoft Azure cluster using Hyper-V generation version 2 virtual machines (VMs). If the installation program detects that the instance type selected for the VMs does not support version 2, it uses version 1 for the deployment.
 
+[id="ocp-4-11-worker-nodes"]
+==== Default AWS and VMware vSphere compute node resources
+Beginning with {product-title} 4.11, the installation program now deploys AWS and VMware vSphere compute nodes with an instance type that provides 4 vCPUs and 16 GB of virtual RAM.
```

Comment:
In AWS we pick the instance type that provides 4 vCPUs and 16GB RAM but for Vsphere, we do not have instance types. We set the vCPUs and RAM ourselves.

Reply:
Thanks for the clarification. In light of that, I believe we do not have to say "how" the node is deployed, but rather simply that they are now deployed with 4 CPU/16 GB RAM.